### PR TITLE
NP-45333-added-publication-note-field

### PR DESCRIPTION
**Pull request by anetteOlli**

### CODE REVIEW

**Review by torarnet — APPROVED** [review]

File: nva-datamodel-java/src/main/resources/publication-ontology.ttl [hunk 1]
```diff
@@ -402,7 +402,12 @@ nva:AdditionalIdentifier a rdfs:Class ;
   rdfs:comment "" ;
   rdfs:isDefinedBy nva: ;
   rdfs:subClassOf nva:Identifier .
-  
+
+nva:PublicationNote a rdfs:Class ;
+  rdfs:comment     "" ;
```
Comment (brinxmat) [inline nva-datamodel-java/src/main/resources/publication-ontology.ttl:407]: `    rdfs:comment "An historical string containing information pertaining to the Cristin verification process"`

and likely `    owl:deprecated "true"^^xsd:boolean`

Comment (anetteOlli) [reply nva-datamodel-java/src/main/resources/publication-ontology.ttl]: Had to set `rdf:type owl:DeprecatedClass ;` as well as you suggestion in order to make it compile. I must admit I have little experience with ontology, can you double check that it is okay?

File: documentation/schema.yaml [hunk 2]
```diff
@@ -273,18 +223,14 @@ referencedSchemas:
...
         type: string
-        example: null
-    example: null
     discriminator:
       propertyName: type
   Architecture:
     required:
     - type
     type: object
-    example: null
```
[the remark this answers is not in the document]
Comment (anetteOlli) [reply documentation/schema.yaml:287]: This schema is generated during compiling. If I compile the schema from main branch the changes also occurs.

**Review by torarnet — APPROVED** [review]

File: nva-datamodel-java/src/main/java/no/unit/nva/model/PublicationNote.java [hunk 3]
```diff
@@ -0,0 +1,11 @@
+package no.unit.nva.model;
+
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+
+
+@JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
+public record PublicationNote( String publicationNoteMessage) {
```
[the remark this answers is not in the document]
Comment (anetteOlli) [reply nva-datamodel-java/src/main/java/no/unit/nva/model/PublicationNote.java]: Annotated with deprecated as well.

File: nva-datamodel-java/src/main/java/no/unit/nva/model/PublicationNote.java [hunk 4]
```diff
@@ -4,8 +4,8 @@
 import com.fasterxml.jackson.annotation.JsonAutoDetect;
 import com.fasterxml.jackson.annotation.JsonTypeInfo;
 
-
+@Deprecated
```
Comment (brinxmat) [inline nva-datamodel-java/src/main/java/no/unit/nva/model/PublicationNote.java:7]: Modig

**Review by brinxmat — APPROVED** [review]

**Pull request merged**